app/repository/repository-detail-controller.js

```diff
@@ -27,13 +34,13 @@ angular.module('repository-detail-controller', ['registry-services', 'app-mode-s
     $scope.getNextHref = function (){
       if($scope.maxTagsPage > $scope.tagsCurrentPage){
         var nextPageNumber = $scope.tagsCurrentPage + 1;
-        return '/repository/'+$scope.repository+'/'+ $scope.tagsPerPage +'/' +nextPageNumber;
+        return '/repository/'+$scope.repository+'?tagsPerPage='+ $scope.tagsPerPage +'&tagPage=' +nextPageNumber;
```

Comment:
From https://www.sitepoint.com/premium/books/angularjs-novice-to-ninja/preview/using-routeparams-in-the-controller-6e0d5e2:

> Apart from named parameters, `$routeParams` also parses the query string, if present. So, if your URL contains a query string like `?key1=value1`, the key/value pair is added to `$routeParams`.

Also see the example at the bottom of https://docs.angularjs.org/api/ngRoute/service/$routeParams:

```javascript
// Given:
// URL: http://server.com/index.html#/Chapter/1/Section/2?search=moby
// Route: /Chapter/:chapterId/Section/:sectionId
//
// Then
$routeParams ==> {chapterId:'1', sectionId:'2', search:'moby'}
```